spec/requests/dashboard_requests_spec.rb

```diff
@@ -34,6 +34,20 @@
         end
       end
     end
+
+    context "BroadcastAnnouncement card" do
+      it "displays announcements if there are valid ones" do
+        BroadcastAnnouncement.create(message: "test announcement", user_id: 1, organization_id: nil)
+        get dashboard_path(default_params)
+        expect(response.body).to include("test announcement")
+      end
+
+      it "doesn't display announcements if they are not from super admins" do
+        BroadcastAnnouncement.create(message: "test announcement", user_id: 1, organization_id: 1)
```

Comment:
Do we have the corresponding check that it doesn't display announcements from super admins on the partners dashboard?

Reply:
Yes, there are two tests on the partner dashboard from the last issue. One of them would fail if admin announcements were being displayed

Reply:
For clarity, I'm looking  for a test that demonstrates that, if there are admin announcements, those announcements do not show in the partner dashboard.  I do not see that test  (in main, in partners/dashboard_requests_spec.rb)

Reply:
I apologize, I was mixed up, there is a test on the model that would fail if an announcement from an organization other than the one passed was returned. I added the test you want